src/parser/compat.ts

```diff
@@ -31,16 +30,22 @@ export function getModuleFromRoot(
 }
 export function getOptionsFromRoot(
   svelteAst: Compiler.Root | SvAST.AstLegacy,
-  code: string,
 ): Compiler.SvelteOptionsRaw | null {
   const root = svelteAst as Compiler.Root;
   if (root.options) {
-    if ((root.options as any).__raw__) {
-      return (root.options as any).__raw__;
-    }
-    // If there is no `__raw__` property in the `SvelteOptions` node,
-    // we will parse `<svelte:options>` ourselves.
-    return parseSvelteOptions(root.options, code);
+    return {
+      type: "SvelteOptions",
+      name: "svelte:options",
+      attributes: root.options.attributes,
```

Comment:
The SvelteOptions node now provides attributes so we don't have to parse it ourselves.